media-video/mpv/mpv-0.12.0.ebuild

```diff
@@ -0,0 +1,258 @@
+DOCS=( README.md etc/example.conf etc/input.conf )
+
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/mpv-player/mpv.git"
+	inherit git-r3
+else
+	SRC_URI+=" https://github.com/mpv-player/mpv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux"
+	DOCS+=( RELEASE_NOTES )
+fi
+
+# See Copyright in source tarball and bug #506946. Waf is BSD, libmpv is ISC.
+LICENSE="GPL-2+ BSD ISC"
+SLOT="0"
+IUSE="+alsa bluray cdio +cli doc-pdf drm dvb +dvd egl +enca encode +iconv
+jack jpeg lcms +libass libav libcaca libguess libmpv lua luajit openal
++opengl oss pulseaudio pvr raspberry-pi rubberband samba sdl selinux v4l vaapi
+vdpau vf-dlopen wayland +X xinerama +xscreensaver xv"
+
+REQUIRED_USE="
+	|| ( cli libmpv )
+	egl? ( opengl )
+	enca? ( iconv )
+	lcms? ( opengl )
+	libguess? ( iconv )
+	luajit? ( lua )
+	opengl? ( || ( wayland X ) )
+	pvr? ( v4l )
+	vaapi? ( X )
+	vdpau? ( X )
+	wayland? ( egl )
+	xinerama? ( X )
+	xscreensaver? ( X )
+	xv? ( X )
+"
+
+COMMON_DEPEND="
+	libav? ( >=media-video/libav-11:0=[encode?,threads,vaapi?,vdpau?] )
+	!libav? ( >=media-video/ffmpeg-2.4.0:0=[encode?,threads,vaapi?,vdpau?] )
+	sys-libs/zlib
+	X? (
+		x11-libs/libX11
+		x11-libs/libXext
+		>=x11-libs/libXrandr-1.2.0
+		opengl? (
+			virtual/opengl
+			egl? ( media-libs/mesa[egl] )
+		)
+		lcms? ( >=media-libs/lcms-2.6:2 )
+		vaapi? ( >=x11-libs/libva-0.34.0[X(+)] )
+		vdpau? ( >=x11-libs/libvdpau-0.2 )
+		xinerama? ( x11-libs/libXinerama )
+		xscreensaver? ( x11-libs/libXScrnSaver )
+		xv? ( x11-libs/libXv )
+	)
+	alsa? ( >=media-libs/alsa-lib-1.0.18 )
+	bluray? ( >=media-libs/libbluray-0.3.0 )
+	cdio? (
+		dev-libs/libcdio
+		dev-libs/libcdio-paranoia
+	)
+	drm? ( x11-libs/libdrm )
+	dvb? ( virtual/linuxtv-dvb-headers )
+	dvd? (
+		>=media-libs/libdvdread-4.1.3
+		>=media-libs/libdvdnav-4.2.0
+	)
+	enca? ( app-i18n/enca )
+	iconv? ( virtual/libiconv )
+	jack? ( media-sound/jack-audio-connection-kit )
+	jpeg? ( virtual/jpeg:0 )
+	libass? (
+		>=media-libs/libass-0.12.1:=[fontconfig]
+		virtual/ttf-fonts
+	)
+	libcaca? ( >=media-libs/libcaca-0.99_beta18 )
+	libguess? ( >=app-i18n/libguess-1.0 )
+	lua? (
+		!luajit? ( || ( =dev-lang/lua-5.1*:= =dev-lang/lua-5.2*:= ) )
+		luajit? ( dev-lang/luajit:2 )
+	)
+	openal? ( >=media-libs/openal-1.13 )
+	pulseaudio? ( media-sound/pulseaudio )
+	rubberband? ( >=media-libs/rubberband-1.8.0 )
+	samba? ( net-fs/samba )
+	sdl? ( media-libs/libsdl2[threads] )
+	v4l? ( media-libs/libv4l )
+	wayland? (
+		>=dev-libs/wayland-1.6.0
+		media-libs/mesa[egl,wayland]
+		>=x11-libs/libxkbcommon-0.3.0
+	)
+"
+DEPEND="${COMMON_DEPEND}
+	${PYTHON_DEPS}
+	virtual/pkgconfig
+	>=dev-lang/perl-5.8
+	dev-python/docutils
+	doc-pdf? ( dev-python/rst2pdf )
+	X? (
+		x11-proto/videoproto
+		xinerama? ( x11-proto/xineramaproto )
+		xscreensaver? ( x11-proto/scrnsaverproto )
+	)
+"
+RDEPEND="${COMMON_DEPEND}
+	selinux? ( sec-policy/selinux-mplayer )
+"
+
+pkg_setup() {
```

Comment:
this info are all misplaced, they belong in `pkg_pretend` which is the only phase which is carried out _before_ the merge. Anything else can be exptected to go unnoticed and the user will not be able to make an informed choice.